tutorials/learnpython.org/en/Modules and Packages.md

```diff
@@ -12,10 +12,13 @@ Modules in Python are simply Python files with a .py extension. The name of the
 A Python module can have a set of functions, classes or variables defined and implemented. 
 In the example above, we will have two files, we will have:
 
-    mygame/
-    mygame/game.py
-    mygame/draw.py
-    
+<li>mygame/</li>

```

Comment:
Please use Markdown syntax (*)

Reply:
Fixed in 5b3549e17dfb91cdaa15b3e40a6420d1f63e8968.